superset/models/dashboard.py

```diff
@@ -171,6 +171,7 @@ def url(self) -> str:
 
     @property
     def datasources(self) -> Set[BaseDatasource]:
+        # pylint: disable=using-constant-test
```

Comment:
do we need this lint disable since there were no changes on this file?

Reply:
yeah it was failing the test, but Hugh merged a fix to it, so I will rebase it.